hw-2/utils.js

```diff
@@ -0,0 +1,12 @@
+const getAutoSuggestUsers = (data = [], loginSubstring = '', limit = null) => {
```

Comment:
As a homework, everything is OK, but in general, please avoid 'utils.js'. autoSuggestUsersService.js is much better name.